addons/mail/static/src/messaging/entity/messaging/messaging_tests.js

```diff
@@ -37,14 +37,8 @@ QUnit.module('Messaging', {
 QUnit.test('currentPartner', async function (assert) {
     assert.expect(7);
 
-    await this.start({
-        session: {
-            name: "Admin",
-            partner_id: 3,
-            partner_display_name: "Your Company, Admin",
-            uid: 2,
-        },
-    });
+    await this.start();
```

Comment:
This test now depends on default session data of mocked messaging service.
Would be nicer to provide session data with `env.session`, so that data are explicit in this test.

Reply:
I think it's fine to test the default data in this case, this test has a very limited use (would be more useful as an integration test with data from a server than a unit test) and `currentPartner` is kind of magic relation itself too.

Reply:
I made the origin of data very obvious by directly comparing to the corresponding session keys. The actual values are irrelevant, only the match is important.